test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -63,21 +63,27 @@ build_branch()
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
     echo "Building branch: \"$1\""
     echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    
+    # Only clone the branch if the option is picked
+    if [ ! -d "build" ]; then
+        git clone --quiet https://github.com/wiredtiger/wiredtiger.git "$1"
+        cd "$1"
+        git checkout --quiet "$1"
```

Comment:
I feel this line is better to be moved down out of the if logic, to deal with cases the "build" directory is previously set up to test a different branch.

Reply:
Maybe line 70 as well?

Reply:
Yes, you're right